mne/io/nirx/nirx.py

```diff
@@ -167,8 +164,11 @@ def __init__(self, fname, saturated, preload=False, verbose=None):
         else:
             if hdr['GeneralInfo']['NIRStar'] not in ['"15.0"', '"15.2"',
                                                      '"15.3"']:
-                raise RuntimeError('MNE does not support this NIRStar version'
-                                   ' (%s)' % (hdr['GeneralInfo']['NIRStar'],))
+                raise RuntimeError(
+                    'MNE does not support this NIRStar version'
+                    ' (%s).' % (hdr['GeneralInfo']['NIRStar'],) +
+                    'We recommend using another software such as `homer3` to '
```

Comment:
Using Homer won't actually work. Their software isn't fully compatible with snirf yet. So I suggest leaving this as it was and say that we only support version 15 and above of nirx files. The versions we support is well described in the documentation

Reply:
Hmm okay well do we have any advice for someone earlier than version 15 nirx files? Ideally, I think it would be nice to point someone in the right direction rather than just saying you can't use MNE. I have no experience with this at all but the earlier version that I did see was almost exactly the same so it's a shame that there would be no path using MNE for them.

Reply:
It looks like `NIRS Toolbox` supports saving out to `snirf` https://github.com/huppertt/nirs-toolbox/blob/4c454acee9feb92e401e4bf8a4027dbe64fec2e8/%2Bnirs/%2Bio/saveSNIRF.m. I think maybe that's worth giving a try for someone who has older data. It requires MATLAB but that's better than not recommending anything.

Reply:
The files produced by the nirs toolbox don't pass the snirf validator. At least that was the case when I checked about 2 months ago. There is a post on the MNE forum about that too. Homer is a better choice than the nirs toolbox, it's more widely used and developed by a larger team. Hopefully Homer will do what we want in the near future. It just doesn't work yet.